containers/Makefile

```diff
@@ -196,23 +191,13 @@ $(SIGNING_SERVICE_SOURCE_FROM_CONCENT_API): build/signing-service/%: build/conce
 	mkdir --parents "build/signing-service/"
 	cp "$<" "$@"
 
-build/verifier/golem/scripts/: build/repositories/golem/apps/rendering/resources/scripts/
-	mkdir --parents "$@"
-	cp  --recursive "$<." "$@"
-
 $(NGINX_STORAGE_SOURCE_FROM_NGINX_PROXY): build/nginx-storage/%: build/nginx-proxy/%
 	mkdir --parents "build/nginx-storage/"
 	cp "$<" "$@"
 
-build/verifier/golem/imgverifier-requirements.txt: build/repositories/golem/apps/rendering/resources/images/requirements.txt
-	cp "$<" "$@"
-
 build/repositories/concent: build/package-builder/clone-or-update.sh /var/concent-data/concent
 	build/package-builder/clone-or-update.sh "concent" "$@"
 
-build/repositories/golem/: build/package-builder/clone-or-update.sh /var/concent-data/golem
-	build/package-builder/clone-or-update.sh  "golem" "$@"
```

Comment:
`clone-or-update.sh` still has code to support the `golem` argument but will no longer work because `golem_version` was removed.

Please remove the first parameter and rename the script to `clone-or-update-concent.sh`.